dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo

```diff
@@ -0,0 +1,89 @@
+within Dynawo.Examples.SMIB;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model SMIB_StepPm "Synchronous machine infinite bus - Step on Pm"
```

Comment:
This is an open question: most of our models in the Modelica library are without "_" but with something like SMIBStepPm? The only "_" are reserved for "_INIT" models at the moment. Do we want to keep to this coding style?

Reply:
you're right, I just named it after dynawo test case. I will rename it